cxx/gendb.hh

```diff
@@ -85,6 +85,18 @@ class GenDB {
           stored_value_map,
       const size_t ind, const bool from_cluster_only);
 
+  // Returns a copy of stored_values, with the items updated to associate
+  // class_name.ref_field at index class_item with new_ref_val. This update is
+  // recursive, such that the reference fields of the class corresponding to
+  // ref_field are updated as well.
```

Comment:
Is this method "philosophically" const?  I think it is, because I think that get_relation_items is const (even though it isn't marked as such) and this only modifies reference_values temporarily.

Anyway, if it is, please state that in the description (and consider making get_relation_items explicitly const).

Reply:
Yes, it behaves as const -- I added a comment, and made get_relation_items const.